drivers/charger/bq24190.h

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
Is this likely to be used any other place than the bq24190.c file? If not then its perfectly fine to move this to the .c file.

Reply:
I could see `drivers/regulator` needing some of these defines for controlling the OTG and SYS rails. I think I'll leave the header as-is if this makes sense.

Reply:
mmh not really, you certainly don't want to include a charger header from a regulator driver. Look at how other MFD drivers are written.

Either way, you may make an argument for the register definitions but the data/config ones should definitely go in the driver file if they don't have to be shared, just to keep the scope under control and for convenience.

Reply:
gotcha, well I think I'll move the data/config structs into `bq24190.c` and keep the defines in `bq24190.h`.